src/main/java/com/example/MyBatisDemo/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.example.MyBatisDemo.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public ResponseEntity<?> studentNOtFoundException(ResourceNotFoundException ex, WebRequest request){
```

Comment:
Please follow the correct naming convention.

Reply:
Done